README.md

```diff
@@ -1,5 +1,25 @@
 # azure-hanaonazure-cli-extension
 
+# Install
```

Comment:
nit: make all of the sections underneath `# azure-hanaonazure-cli-extension` a smaller font: i.e., `## Install`

Reply:
I'm not sure I agree with this suggestion. ```# azure-hanaonazure-cli-extension```is top level header (h1), your suggestion imply that everything else is a 2level header (h2). I would rather remove completely that header since I don't see the value it provides. Either way, I think it's out of the scope of this PR, if we change headers or delete the first one, I will do it on a new PR.

Reply:
ack